src/Analysis/GWASV2/Components/Covariates/CustomDichotomousCovariates.jsx

```diff
@@ -80,23 +78,23 @@ const CustomDichotomousCovariates = ({
         <div data-tour='choosing-dichotomous'>
           <div className='GWASUI-flexRow' data-tour='table-repeat'>
             <div>
-              <h3>Select NO Cohort</h3>
-              <CohortSelect
+              <h3>Get Value 0</h3>
+              <SelectCohortDropDown
                 selectedCohort={firstPopulation}
                 handleCohortSelect={setFirstPopulation}
               />
             </div>
             <div>
-              <h3>Select YES Cohort</h3>
-              <CohortSelect
+              <h3>Get Value 1</h3>
+              <SelectCohortDropDown
                 selectedCohort={secondPopulation}
                 handleCohortSelect={setSecondPopulation}
```

Comment:
can this be more generic?

Reply:
if not i can do it in my pr

Reply:
IMO this is a good version for this PR. Let's discuss further improvements in a follow-up PR or ticket